### DISCUSSION TITLE
ATX12VO Branch

### DISCUSSION THREAD

**Discussion by TheGuyDanish** [TheGuyDanish/CM4_MATX#46] [Ideas]
Converted to a discussion to clean up issues.

Continues from #35

Summary: An ATX12VO branch could be an interesting piece of work to put into the project, but given that currently, ATX12VO hasn't spread among DIY builders, the likes of which are likely to use a board like this, making a separate branch for it already seems too hasty.

Will revisit this once Revision 1.0 is achieved.

**Comment by DanForever** [comment]
The first question that popped up in my mind when I discovered this project was "I wonder if there'll be a 12VO only variation?"

I'm glad to see I'm not the first 😂

**Comment by erkinalp** [comment]
With an ATX12VO variant, I could use existing HP Elite case and power supply. Well, because we do not need a large footprint for this board.
